wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/DeveloperToolsView.swift

```diff
@@ -60,8 +60,12 @@ struct DeveloperToolsView: View {
 
         case let .text(textItem):
             TextItemCell(title: textItem.title, value: textItem.value)
-                .onTapGesture {
-                    viewModel.handleEvent(.itemTapped(item))
+                .contextMenu {
+                    SwiftUI.Button {
+                        viewModel.handleEvent(.itemTapped(item))
```

Comment:
I suggest to rename the case `itemCopyRequested`

Reply:
Done!

Reply:
I mean, `viewModel.handleEvent(.itemCopyRequested(item))`

Reply:
Done! 🎉